src/components/Tooltip/index.tsx

```diff
@@ -41,24 +41,22 @@ export const Tooltip = (props: TTooltipProps): JSX.Element | null => {
         }}
         ref={setElemRef}
       >
-        {isHovered && (
-          <div
-            data-id={dataId}
-            className={`tooltip tooltip--${size} tooltip--${tooltipPosition} ${className}`}
-            ref={setTooltipRef}
+        <div
+          data-id={dataId}
+          className={`tooltip tooltip--${size} tooltip--${tooltipPosition} ${className}`}
+          ref={setTooltipRef}
+        >
```

Comment:
where is isHovered?

Reply:
Like a suggestion: use classnames library to build className